### ISSUE TITLE
mtu isn't being obeyed (even with the option enabled in dhcpcd.conf)

### ISSUE THREAD

**Issue by prometheanfire** [prometheanfire/gentoo-cloud-prep#15]

**Comment by hartwork** [comment]
It's disabled by default:

```
# Some interface drivers reset when changing the MTU so disabled by default.
#option interface_mtu
```

I don't see it being enabled yet. Would expect to see such at
https://github.com/prometheanfire/gentoo-cloud-prep/blob/master/files/prep.sh#L33 .

**Comment by prometheanfire** [comment]
seems like this is fixed in dhcpcd-9999, 6.9.[0,1] resets the mtu to 1500 for some reason.  6.9.2 seems to be needed (unreleased)

**Comment by prometheanfire** [comment]
https://bugs.gentoo.org/555974

**Comment by prometheanfire** [comment]
not fixed in 9999

**Comment by prometheanfire** [comment]
there is now a fix in master

http://roy.marples.name/projects/dhcpcd/ci/5e375e2d3180d184?sbs=0

**Comment by prometheanfire** [comment]
fixed, 6.9.2 is out, images re-built and uploaded
